src/h3/api/basic_int/__init__.py

```diff
@@ -996,3 +996,70 @@ def great_circle_distance(latlng1, latlng2, unit='km'):
         lat2, lng2,
         unit = unit
     )
+
+
+def cell_to_vertex(h, vertex_num):
+    """
+    Return a (specified) vertex of an H3 cell.
+
+    Parameters
+    ----------
+    h : H3Cell
+    vertex_num : int
+        Vertex number (0-5)
+
+    Returns
+    -------
+    The vertex
+    """
+    return _cy.cell_to_vertex(_in_scalar(h), vertex_num)
+
+
+def cell_to_vertexes(h):
+    """
+    Return a list of vertices of an H3 cell. The list will always be of length 6.
+    If a pentagon is entered, the last element will be 0.
```

Comment:
In the Python world, I'd rather have us dynamically size the array: a hex should return an array of length 6, and a pentagon should return an array of length 5.

Reply:
Yeah I fixed array length to 6 only to align with the official docs... I'll change this.